tests/handlers.js

```diff
@@ -17,6 +18,57 @@ const api = path => {
     return `https://api.groupme.com/v3${path}`
 }
 
+function wrapJsonBody(status, body, errors) {
+    const jsonBody = {
+        meta: {
+            code: status,
+        },
+        response: body
+    }
+    if(errors !== undefined) jsonBody.meta.errors = errors
+    
+    return jsonBody
+}
+
+function constructHandler(reqType, path, status, body, bodyType = 'json', errors, headers) {
+    const requestResolver = async (req, res, ctx) => {
+        const resParams = []
```

Comment:
(just tacking this on to the other suggestion so you can accept/reject both as you please)

```suggestion
        let resStatus, resBody, resHeaders
```

Reply:
The response headers parameter is optional though, so it's not guaranteed to be set.